engine/runtime-integration-tests/src/test/java/org/enso/interpreter/node/callable/SequenceLiteralNodeTest.java

```diff
@@ -6,17 +6,18 @@
 import org.enso.interpreter.node.expression.literal.LiteralNode;
 import org.enso.interpreter.runtime.error.PanicException;
 import org.enso.interpreter.runtime.error.PanicSentinel;
-import org.enso.test.utils.ContextUtils;
+import org.enso.test.utils.ContextRule;
+import org.junit.ClassRule;
 import org.junit.Test;
 
 public class SequenceLiteralNodeTest {
+  @ClassRule public static final ContextRule ctxRule = ContextRule.createDefault();
```

Comment:
There should be no need for `ctxRule.executeInContext(this::propagatePanicSentinelImpl);`:
- either the code should already run in the context
- or one should be able to request such behavior

```suggestion
  @ClassRule public static final ContextRule ctxRule = ContextRule.newBuilder().runInContext(true).build();
```

Reply:
Done in f5e502daf80a71450bda240b2a0933314512f901